commit 66f991ad90b6cbb93a2c5ef9f5f83d745b06a962
Author: Richard Hughes <richard@hughsie.com>
Date:   Wed Apr 9 01:28:38 2008 +0100

    don't show the refresh menuitem if we are doing an update-packages call either

diff --git a/src/gpk-watch.c b/src/gpk-watch.c
index e4adc1c..ee7b152 100644
--- a/src/gpk-watch.c
+++ b/src/gpk-watch.c
@@ -244,6 +244,7 @@ gpk_watch_task_list_changed_cb (PkTaskList *tlist, GpkWatch *watch)
 	g_return_if_fail (GPK_IS_WATCH (watch));
 
 	if (pk_task_list_contains_role (tlist, PK_ROLE_ENUM_REFRESH_CACHE) ||
+	    pk_task_list_contains_role (tlist, PK_ROLE_ENUM_UPDATE_PACKAGES) ||
 	    pk_task_list_contains_role (tlist, PK_ROLE_ENUM_UPDATE_SYSTEM)) {
 		watch->priv->show_refresh_in_menu = FALSE;
 	} else {
